aptos-move/framework/src/generated/mod.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod aptos_framework_sdk_builder;
+pub mod aptos_stdlib;
+pub mod aptos_token_sdk_builder;
```

Comment:
Let's try to keep the refactors in separate commits next time.  Makes it easier to check the things that have proper new code changes.

Reply:
It is impossible for such a refactor, or it will take 10x more time. You can't design this top-down. There is so many legacy code which uses code and calls the Move compiler in some strange ways (mostly tests) that the design needed to be revised multiple times to satisfy all requirements.

Reply:
Oh I misunderstood you. You meant multiple commits in one PR, and you are right, this can be done better. To be honest I haven't really figured out how to comfortably rebase with a series of commits, that is why I'm reluctant. At FB and at Google, you push a button.